app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
fix tương tự bên dưới
```suggestion
  before_action :logged_in_user, only: %i(index edit update destroy)
```